dvtemplates/new_cdi_image.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+error_message() {
+    OS=$2
+    if [ $1 -eq 1 ]; then
+        echo -e "Unable to access data from the OS release URL"
```

Comment:
Can we assign these integers to named constants? Would help readability when used.

Reply:
Done